### PR TITLE
[FEATURE] Subtask - Make context implicit

### PR DESCRIPTION
**Pull request by MatthewK3023**

### CODE REVIEW

**Review by NoctisHong — CHANGES_REQUESTED** [review]
After making Context implicit, you need to update all example files.

**Review by NoctisHong — CHANGES_REQUESTED** [review]
After making Context implicit, you also need to integrate the doc of Context to Client

File: examples/jupyter_example.ipynb [hunk 1]
```diff
@@ -63,11 +63,22 @@
...
+    "# enable default logger\n",
+    "core.enable_default_logger()"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note: It is ok to recreate context to set the usr, pwd, host."
+    "### Create Decanter Core Client\n",
```
Comment (schwannden) [inline examples/jupyter_example.ipynb:80]: why is there a \n here?

File: examples/example.py [hunk 2]
```diff
@@ -19,14 +19,13 @@ def main():
     # Enable default logger for logging message
     core.enable_default_logger()
 
+    # The Main object handles the calling of Decanter's API.
+    client = core.CoreClient()
```
Comment (schwannden) [inline examples/example.py:23]: no need to add create method, just supply username, password, host as construct parameter here, and connect and create context in class constructor

**Review by schwannden — DISMISSED** [review]

**Review by NoctisHong — DISMISSED** [review]

**Review by NoctisHong — APPROVED** [review]

**Pull request merged**